pages/popular.js

```diff
@@ -29,12 +29,14 @@ export default function Popular({ data }) {
         <link rel="icon" href="/favicon.ico" />
       </Head>
 
-      <div className="mx-auto container">
-        <h1 className="text-4xl font-bold mb-4">Popular Profiles</h1>
-        <ul>
+      <div className="p-5">
+        <h1 className="lg:text-3xl md:text-2xl text-xl font-bold mb-4 w-fit p-3 text-[#202328] rounded border-b-4 border-b-green-500">
+          Popular Profiles
+        </h1>
+        <ul className="flex flex-wrap gap-3 justify-center mt-[3rem]">
           {data.map((profile) => (
             <li key={profile.username}>
-              <UserPreview profile={profile} />
+              <UserCard profile={profile} />
```

Comment:
It would be good to have an on hover effect similar to before and to be consistent with the rest of the app